cartography/intel/gcp/cloudmonitoring.py

```diff
@@ -28,6 +29,8 @@ def get_monitoring_alertpolicies(monitoring: Resource, project_id: str) -> List[
                     policy['id'] = policy['name']
                     policy['policy_name'] = policy.get('name').split('/')[-1]
                     policy['labels'] = policy.get('userLabels', {})
+                    policy['consoleLink'] = gcp_console_link.get_console_link(project_id=project_id,\
+                        alert_policy_name=policy['name'], resource_name='cloud_monitoring_alert_policy')
```

Comment:
we should pass `policy_name ` key as a `alert_policy_name `

Reply:
We are already splitting the policy name in console link project.